### PR TITLE
ECO-5367 - Edits to the README (and other docs)

### PR DESCRIPTION
**Pull request by jeffswartz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -223,238 +260,369 @@ To enable this feature:
...
-- `CHROME_EXTENSION_ID` :(Optional, LEGACY, default value: 'null'): The Chrome AddOn extension ID for screen sharing. Note: the Chrome extension is no longer required for screensharing.
-- `ENABLE_ANNOTATIONS`: (Optional, default value: true) Whether to enable annotations in screen sharing. This is only meaningful if screen sharing is not disabled.
+To enable and configure screen sharing, edit options in the config/config.json file
+or set environment variables:
+
+* `Screensharing.enabled` (config.json) / `ENABLE_SCREENSHARING` (environment variable) -- Whether
+  to enable screen sharing. The default value is `false`.
+
+* `Screensharing.chromeExtensionId` (config.json) / `CHROME_EXTENSION_ID`
+  (environment variable) -- The Chrome add-on extension ID for screen sharing.
```
Comment (rhainer) [inline README.md:411]: I don't think this is needed anymore. Chrome now supports screensharing without it. The only relevant parameter is ENABLE_SCREENSHARING. I think.

File: README.md [hunk 2]
```diff
@@ -223,238 +260,369 @@ To enable this feature:
...
+and configure this, edit options in the config/config.json file or set environment variables:
 
-The app lets the developer POST feedback data to an endpoint on your HTTP server:
+* `Feedback.url` (config.json) / `FEEDBACK_URL` (environment variable) -- The URL to send a POST
+  request with feedback data. Leave this as an empty string or undefined to disable issue reporting.
+  The default value is ''.
 
-`FEEDBACK_URL`: The URL to send a POST request with feedback data. Leave this as an empty string or
-undefined to disable issue reporting.
+* `Feedback.reportIssueLevel` (config.json) / `REPORT_ISSUE_LEVEL` (environment variable) -- The audio
```
Comment (rhainer) [inline README.md:454]: This section likely needs to be cleaned up now that we implemented HotJar. I'm not sure what it should say.

**Review by rhainer — COMMENTED** [review]
Excellent updates. Just a few minor comments.

File: README.md [hunk 3]
```diff
@@ -72,144 +109,144 @@ Edit `config/config.json` and replace `<key>` and `<secret>` with your OpenTok A
...
-    },
-    "Archiving": {
-        "enabled": true,
-        "archiveManager": {
-            "enabled": true
-        }
-    }
-}
-```
+You can also copy the existing `config/example.json` to `config/config.json` file.
```
[the remark this answers is not in the document]
Comment (rhainer) [reply README.md:112]: Also, make the config/example.json text a link to the file. That way If I want to see an example, I can just click.

File: README.md (hunk 1 above)
Comment (jeffswartz) [reply README.md:411]: A screen-sharing extension is required by older versions of Chrome and Opera (which we still support). I'll make the README docs clearer.

File: README.md (hunk 2 above)
Comment (jeffswartz) [reply README.md:454]: I added some info on Hotjar. WRT reportIssueLevel, it is unused by Hotjar. I'll make that clear in the README.

**Review by rhainer — APPROVED** [review]
lgtm

**Pull request merged**